packages/math/src/pool/__tests__/stable.spec.ts

```diff
@@ -1,89 +1,362 @@
-import * as StableMath from "../stable";
-import { Coin, Dec } from "@keplr-wallet/unit";
+import * as StableMath from '../stable';
+import { Coin, Dec, Int, DecUtils } from '@keplr-wallet/unit';
 
-describe("Test stableswap math", () => {
-  describe("calcOutGivenIn", () => {
-    test("even pool basic trade", () => {
+describe('Test stableswap math', () => {
+  describe('calcOutGivenIn', () => {
+    test('even pool basic trade', () => {
       const poolAssets: StableMath.StableSwapToken[] = [
         {
           amount: new Dec(1_000_000_000),
-          denom: "foo",
+          denom: 'foo',
```

Comment:
Hmm it seems your vscode setup (prob prettier) is changing double quotes to single

My only hunch right now is you're missing eslint? Try installing this extension below and see if that fixes it. If not, DM me.

https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint